napari/utils/translations.py

```diff
@@ -629,21 +683,19 @@ def _load_language(
                 import warnings
 
                 warnings.warn(
-                    "The `language` setting defined in the napari "
-                    "configuration file could not be read. The "
-                    f"default language will be used.\n\nError: \n{err}"
-                )
-
+                    f"The `language` setting defined in the napari configuration file could not be read.\n\nThe default language will be used.\n\nError:\n{err}"
+                ),
```

Comment:
if not translated can we leave it as a f-string ? or maybe you do that to avoid error from your script I guess...

Reply:
Fixed!